src/bootstrap/src/core/build_steps/compile.rs

```diff
@@ -1624,42 +1624,16 @@ impl Step for Sysroot {
             });
         }
 
-        // Symlink the source root into the same location inside the sysroot,
+        // Copy the source root into the same location inside the sysroot,
         // where `rust-src` component would go (`$sysroot/lib/rustlib/src/rust`),
         // so that any tools relying on `rust-src` also work for local builds,
         // and also for translating the virtual `/rustc/$hash` back to the real
         // directory (for running tests with `rust.remap-debuginfo = true`).
```

Comment:
I am not sure if this is going to change.

Reply:
Future note for worst-case scenario: We can make this to use copying for CI environments and create symbolic links for non-CI environments.